docs/analysis-techniques.md

```diff
@@ -43,29 +43,29 @@ If this function could be naively tested, a test could be created to only check
 
-Note that the the term 'source code' was used here. This can either the be actual source code or the compiled bytecode. The reason for pointing this out if because there are also non-automated verification techniques. For these techniques the user has to define a model of how to design works (the design, not the code). Additionally, properties on this design have to be defined. The verification tooling will then take the model and the properties and checks if they can be satisfied.
+Note that the term 'source code' was used here. This can either the be actual source code or the compiled bytecode. The reason for pointing this out is that there are also non-automated verification techniques. For these techniques, the user has to define a model of how to design works (the design, not the code). Additionally, the properties of this design have to be defined. The verification tooling will then take the model and the properties and check if they can be satisfied.
 
 What you will notice when diving more into these different (non-)automated tooling is that there will always be a bit of both, while there are clear differences between techniques and tooling, there is also quite some overlap.
 
-One of the techniques used as the backbone of both non-automated and automated technique is [Satisfiable Modulo Theory (SMT)](#satisfiable-modulo-theory-smt). The main purpose of SMT is to check if the variables in a program can have a certain (initial) value such that a requirement is met. In other words, if there is a **satisfiable** assignment for the variables.
+One of the techniques used as the backbone of both non-automated and automated techniques is [Satisfiable Modulo Theory (SMT)](#satisfiable-modulo-theory-smt). The main purpose of SMT is to check if the variables in a program can have a certain (initial) value such that a requirement is met. In other words, if there is a **satisfiable** assignment for the variables.
 
-A technique that uses SMT is [Symbolic execution](#symbolic-execution) that checks all branches of a program to see if any of them lead to a failing assertions. If it finds a failing branch it will check if that branch can be reached using SMT. But SMT is also used during other stages in symbolic execution. For example to avoid wasting processing time on branches that can't be reached anyway. In other words to prune (remove) these branches from the analysis process.
+A technique that uses SMT is [Symbolic execution](#symbolic-execution) which checks all branches of a program to see if any of them lead to a failing assertion. If it finds a failing branch it will check if that branch can be reached using SMT. But SMT is also used during other stages in symbolic execution. For example to avoid wasting processing time on branches that can't be reached anyway. In other words to prune (remove) these branches from the analysis process.
```

Comment:
[Symbolic execution](#symbolic-execution) link does not seem to work

Reply:
[afac119](https://github.com/Byont-Ventures/smart-contract-analysis-tools/pull/47/commits/afac1198df618d42b4f78a0b408d8372ecdcde8a)